README.md

```diff
@@ -43,11 +43,15 @@ fail during the verification phase if you don't follow them.
 
     mvn clean package
     
-Run findbugs for future reference
-or to make sure you haven't introduced any new warnings
+Run findbugs for future reference or to make sure you haven't introduced any
+new warnings
 
     mvn findbugs:findbugs
-    
+
+Run checkstyle
+
+    mvn checkstyle:checkstyle
```

Comment:
```checkstyle:checkstyle``` is run during the compile phase and ```checkstyle:check``` is run during the verify phase, so there is not need to run it again.